### DISCUSSION TITLE
Authenticating a user from react to nodes to BB and back

### DISCUSSION THREAD

**Discussion by aviNrich** [Budibase/budibase#12576] [Help] [answered]
Hi,
I have a website written in react and nodejs. now I want to onboard users to the website and let them login and save some data about them. the BB will be opened only for the site admins.

So I know I can create users via: https://budibase.app/api/public/v1/users
Now, in my react app, the user needs to login, I want to send a login request to my nodejs server and then send the login request to BB.
how can I do that? is BB built for this use-case? I mean onboard users that will not be logged in to BB
if its possible, how can I remotely authenticate and give him his jwt token.

Thanks

**Comment by ConorWebb96** [comment, marked as answer]
Hey @aviNrich,

I don't believe this is something which is possible. You can obviously create an account and login the user through the public api within Budibase itself. Here is an example of [this](https://github.com/Budibase/budibase/discussions/6098). 

However, Its the storing the of the token and using this in an external codebase which I believe you won't be able to do.

In short I don't believe you can remotely authenticate in the way you are describing.

I hope this helps.
